avogadro/qtplugins/overlayaxes/overlayaxes.cpp

```diff
@@ -5,19 +5,19 @@
 
 #include "overlayaxes.h"
 
+#include <avogadro/qtopengl/glwidget.h>
```

Comment:
This is the cause of the compile failure. We just need to move this down after `avogadrogl.h` somewhere and make sure clang-format doesn't swap them again.

Reply:
Thank you for your check. I reverted the sort.